src/containers/home.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import Home from "../components/home.js";
+import { useQuery } from "@apollo/react-hooks";
+import { EXCHANGE_RATES } from "../queries/foo";
+
+function HomeContainer() {
+  const { loading, error, data } = useQuery(EXCHANGE_RATES);
```

Comment:
I know this is a PoC but the constant `EXCHANGE_RATES` doesn't make a ton of sense in this context. Probably should use something like `RESOURCES`?

Reply:
Make sense 👍